### PR TITLE
[sailfish-browser] Update Aptoide installation support. Fixes JB#32823

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by rojkov** [comment]
LGTM

**Comment by rainemak** [comment]
Rebased and cleaned a bit.

File: src/downloadmanager.cpp [hunk 1]
```diff
@@ -166,21 +176,6 @@ bool DownloadManager::moveMyAppPackage(QString path)
     return true;
 }
 
-QString DownloadManager::aptoideApk(QString packageName)
-{
-    QString apkPath("/data/app/");
-    QString aptoideApk = QString("%1/%2.apk").arg(apkPath, packageName);
-    if (!QFile(aptoideApk).exists()) {
```
Comment (rainemak) [inline src/downloadmanager.cpp]: I don't see any reason for this kind of lightweight Aptoide installed check.

As ```apkd-launcher com.aptoide.partners.jolla_tablet_store-1.apk``` works without path we should not have path there.

File: src/downloadmanager.cpp [hunk 2]
```diff
@@ -87,38 +95,41 @@ void DownloadManager::recvObserve(const QString message, const QVariant data)
...
-            if (apkName.isEmpty()) {
-                qWarning() << "No aptoide client installed to handle package: " + targetPath;
-                return;
-            }
+            QString rootNameSpace("com.aptoide.partners%1");
+            QString aptoideSupport = rootNameSpace.arg(QLatin1Literal(".AptoideJollaSupport"));
+#if CPU_ARM
+            QString packageName = rootNameSpace.arg(QString());
+#else
+            QString packageName = rootNameSpace.arg(QLatin1Literal(".jolla_tablet_store"));
```
Comment (rojkov) [inline src/downloadmanager.cpp]: I guess there might be semantic problems in case of ARM based tablet.

Comment (rainemak) [inline src/downloadmanager.cpp]: Yeap. That needs to be tackled later. Currently ```tablet``` package implies X86 arch apks.

**Comment by rojkov** [comment]
LGTM,
Successfully tested on the tablet.

**Pull request merged**